apps/mobile/src/features/send/send-sheet-content.tsx

```diff
@@ -0,0 +1,21 @@
+import { useSendContext } from './send-context';
+import { SelectAccount } from './send-steps/select-account';
+import { SelectAsset } from './send-steps/select-asset';
+import { SendForm } from './send-steps/send-form';
+
+export function SendSheetContent() {
+  const { step } = useSendContext();
+
+  switch (step) {
```

Comment:
This should have type `SendStep` right?

Reply:
This is already typed here.

Reply:
OK, I wasn't sure if it needed to be re-added in the switch. Thanks! 🙇